substratevm/NativeImageMavenPlugin.md

```diff
@@ -25,7 +25,7 @@ separate step. It is sufficient to add `native-image-maven-plugin` into the
         <skip>false</skip>
         <imageName>example</imageName>
         <buildArgs>
-            --no-fallback
+            <buildArg>--no-fallback</buildArg>
```

Comment:
Please keep those examples as they are. We want to show that it is possible to have individual options in separate lines.

Reply:
Also it would be nice to explain how to do reusability from a parent pom as you show in https://github.com/oracle/graal/issues/2950#issue-733337873

Reply:
Done.